### PR TITLE
Display shadow effect when hover.

### PR DESCRIPTION
**Pull request by SamriddhiMishra**
Fixes #https://github.com/CircuitVerse/Interactive-Book/issues/20

#### Describe the changes you have made in this pr -
Changed style to give a shadow effect.
### Screenshots of the changes (If any) -
Before:-
Before

After:-
After

### CODE REVIEW

File: app/assets/stylesheets/logixRelease.css [hunk 1]
```diff
@@ -129,4 +129,8 @@ a.btn.btn-outline-primary {
 
 a.btn.btn-outline-danger {
     margin-top: 10px;
-}
\ No newline at end of file
+}
+
+.card-shadow:hover {
+    box-shadow: 4px 4px #dfdfdf;
+}
```
Comment (snitin315) [inline app/assets/stylesheets/logixRelease.css]: Add a smooth ease-in transition as well for this hover effect.

**Comment by aayush-05** [comment]
Looks good, but I am not very sure if this can be merged now since the home page has been redone under new UI @nitin10s

File: app/assets/stylesheets/logixRelease.css (hunk 1 above)
Comment (SamriddhiMishra) [reply app/assets/stylesheets/logixRelease.css]: After2

File: app/assets/stylesheets/logixRelease.css [hunk 2]
```diff
@@ -129,4 +129,9 @@ a.btn.btn-outline-primary {
 
 a.btn.btn-outline-danger {
     margin-top: 10px;
-}
\ No newline at end of file
+}
+
+.card-shadow:hover {
+    box-shadow: 4px 4px #dfdfdf;
+    transition: box-shadow 0.4s ease-in;
```
Comment (snitin315) [inline app/assets/stylesheets/logixRelease.css]: I guess transform property is used with `psedu selectors` like `:hover` , transition property is used with `.class-name`

Comment (Shivansh2407) [reply app/assets/stylesheets/logixRelease.css]: > I guess transform property is used with `psedu selectors` like `:hover` , transition property is used with `.class-name`

True 👍 . Please make the requested changes @SamriddhiMishra .

Comment (SamriddhiMishra) [reply app/assets/stylesheets/logixRelease.css]: Sorry! It was working fine with :hover so I made the changes.
After3

But I will change that in a minute.

Comment (SamriddhiMishra) [reply app/assets/stylesheets/logixRelease.css]: My bad. Thanks!

**Comment by Ananthavijay** [comment]
This has not been implemented in the New UI

**Comment by Shivansh2407** [comment]
I can work on this if not done yet 👍🏻

**Comment by aayush-05** [comment]
Can be closed for now @tachyons

**Pull request closed without merging**